proto/tikvpb.proto

```diff
@@ -58,6 +58,78 @@ service Tikv {
     // transaction debugger commands.
     rpc MvccGetByKey(kvrpcpb.MvccGetByKeyRequest) returns (kvrpcpb.MvccGetByKeyResponse) {}
     rpc MvccGetByStartTs(kvrpcpb.MvccGetByStartTsRequest) returns (kvrpcpb.MvccGetByStartTsResponse) {}
+
+    rpc BatchCommands(stream BatchCommandsRequest) returns (stream BatchCommandsResponse) {}
+}
+
+message BatchCommandsRequest {
+    repeated Request requests = 1;
+    repeated uint64 request_ids = 2;
+    
+    message Request {
+        oneof cmd {
+            kvrpcpb.GetRequest Get = 1;
+            kvrpcpb.ScanRequest Scan = 2;
+            kvrpcpb.PrewriteRequest Prewrite = 3;
+            kvrpcpb.CommitRequest Commit = 4;
+            kvrpcpb.ImportRequest Import = 5;
+            kvrpcpb.CleanupRequest Cleanup = 6;
+            kvrpcpb.BatchGetRequest BatchGet = 7;
+            kvrpcpb.BatchRollbackRequest BatchRollback = 8;
+            kvrpcpb.ScanLockRequest ScanLock = 9;
+            kvrpcpb.ResolveLockRequest ResolveLock = 10;
+            kvrpcpb.GCRequest GC = 11;
+            kvrpcpb.DeleteRangeRequest DeleteRange = 12;
+
+            kvrpcpb.RawGetRequest RawGet = 13;
+            kvrpcpb.RawBatchGetRequest RawBatchGet = 14;
+            kvrpcpb.RawPutRequest RawPut = 15;
+            kvrpcpb.RawBatchPutRequest RawBatchPut = 16;
+            kvrpcpb.RawDeleteRequest RawDelete = 17;
+            kvrpcpb.RawBatchDeleteRequest RawBatchDelete = 18;
+            kvrpcpb.RawScanRequest RawScan = 19;    
+            kvrpcpb.RawDeleteRangeRequest RawDeleteRange = 20;
+            kvrpcpb.RawBatchScanRequest RawBatchScan = 21;
+
```

Comment:
maybe should add `UnsafeDestroyRange` too?

Reply:
This interface won't be called very frequently, right? If so I prefer keep it outside from batch, to make the `service/kv.rs` more simple.